data/test.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<!-- open as resource://ipfs-firefox-addon-at-lidel-dot-org/data/test.html -->

```

Comment:
Cool! It probably should be used in a dedicated  test suite (`tests/test-linkify.js`).

So far I did manual test and got weird results: 

0. Run via `jpm -b vendor/firefox-45.0a2/firefox run`
1. I enabled at addon's `about:addons` screen: 
![2016-01-01-182616_256x49_scrot](https://cloud.githubusercontent.com/assets/157609/12071618/2bcaddfc-b0b5-11e5-9a46-f8c1c1803982.png)
2. Visited `resource://ipfs-firefox-addon-at-lidel-dot-org/data/test.html`
3. Text links were not converted into `<a>` tags. The only change I was able to spot was the `href` of the "abs link", which changed from 
`fs:/` to `https://ipfs.io/`

I did not dig into the underlying code: perhaps it is still work in progress?
Or did I found a bug? :-)